projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -496,4 +512,91 @@ public function get_file( $request ) {
 			)
 		);
 	}
+
+	/**
+	 * Get the status of a forms integration.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function get_integration_status( WP_REST_Request $request ) {
+		$slug = $request->get_param( 'slug' );
+
+		switch ( $slug ) {
+			case 'akismet':
+				return $this->get_akismet_status();
+
+			case 'creative-mail':
+			case 'jetpack-crm':
+				return $this->get_plugin_status( $slug );
+
+			default:
+				return new WP_Error(
+					'invalid_integration',
+					/* translators: %s: integration slug */
+					sprintf( __( 'Unknown integration: %s', 'jetpack-forms' ), $slug )
+				);
+		}
+	}
+
+	/**
+	 * Get basic plugin status (installed/active).
+	 *
+	 * @param string $plugin_slug The plugin slug (e.g. 'akismet' or 'creative-mail').
+	 * @return array Plugin status data.
+	 */
+	private function get_plugin_status( $plugin_slug ) {
+		if ( ! function_exists( 'get_plugins' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+
+		// WordPress core functions like get_plugins() and is_plugin_active() require the full plugin path
+		// (e.g. 'akismet/akismet.php'). Since we expose simpler slugs in our API, we maintain a mapping
+		// of slugs to their full plugin paths. You'll need to update this mapping as you add new integrations.
+		$plugin_files = array(
+			'akismet'       => 'akismet/akismet.php',
+			'creative-mail' => 'creative-mail-by-constant-contact/creative-mail-by-constant-contact.php',
+			'jetpack-crm'   => 'zero-bs-crm/ZeroBSCRM.php',
+		);
+
+		$plugin_file = isset( $plugin_files[ $plugin_slug ] ) ? $plugin_files[ $plugin_slug ] : '';
+		if ( empty( $plugin_file ) ) {
+			return array(
+				'type'        => 'plugin',
+				'isInstalled' => false,
+				'isActive'    => false,
+				/* translators: %s: plugin slug */
+				'error'       => sprintf( __( 'Unknown plugin: %s', 'jetpack-forms' ), $plugin_slug ),
+			);
+		}
+
+		$installed_plugins = get_plugins();
+		$is_installed      = isset( $installed_plugins[ $plugin_file ] );
+		$is_active         = is_plugin_active( $plugin_file );
+
+		return array(
```

Comment:
noticed that for `get_akismet_status` you use `rest_ensure_response` while here you don't

Reply:
Added!